src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -28,7 +28,8 @@ CAC.Control.DirectionsList = (function (_, $, ShareModal, MapTemplates) {
             googlePlusShareButton: '.modal-list-google',
             shareModalButton: '.share-directions',
             stepByStepClass: 'body-step-by-step',
-            sidebarBannerClass: 'body-has-sidebar-banner'
+            hasSidebarBannerClass: 'body-has-sidebar-banner',
```

Comment:
It's kind of a weird one in that it's a class that gets added to the body div to control some height styles on the sidebar.  I think having `sidebarBannerClass` right next to `activeSidebarBanner` made it strike me that `sidebarBannerClass` sounds like a class that identifies a sidebar banner when in fact it's something completely different.  But I don't feel strongly about it.  I'll change it back.